packages/core/src/index.ts

```diff
@@ -174,3 +181,22 @@ export async function processManifest(
       .map((scriptLocation: any) => injectScript(appName, scriptLocation as string, true))
   );
 }
+
+export async function asyncLoader(scope: string, module: string): Promise<{ default: any }> {
+  if (typeof scope === 'undefined' || scope.length === 0) {
+    throw new Error("Scope can't be undefined or empty");
+  }
+  if (typeof module === 'undefined' || module.length === 0) {
+    throw new Error("Module can't be undefined or empty");
+  }
+
+  if (!module.startsWith('./')) {
+    console.warn(`Your module ${module} doesn't start with './' this indicates and error`);
```

Comment:
this indicates and error?

Reply:
Yes. Webpack requires leading `./`

Reply:
Yeah, I just mean that the sentence does not make sense to me 😄 .. shouldn't it be 'this indicates error?'

Reply:
Yeah, typo :smile: